interchain-exchange/tutorial/09-walkthrough.md

```diff
@@ -63,17 +64,21 @@ interchanged q ibcdex list-buyOrderBook --node tcp://localhost:26659
 
 ## Create an Orderbook
 
-Create an orderbook for a new pair of token to the exchange. 
+<!-- Create an orderbook for a new pair of token to the exchange. 
 Source blockchain is `Mars`, target blockchain is `Venus`.
 
-The orderbook is to sell `mcx` and buy `vcx` token.
+The orderbook is to sell `mcx` and buy `vcx` token. -->
+
+Let's create orderbooks for a pair of tokens.
```

Comment:
As recent standards please don't use "we" or "let us" but directly address the reader "you".